lib/assets/data_transform_cell/main.css

```diff
@@ -55,6 +55,10 @@
   --beige-100: #FBF5F1;
   --beige-200: #F8EBE4;
   --beige-600: #EDCDBB;
+
+  --indigo-100: #e0e7ff;
```

Comment:
We're using colors that don't conflict with the Livebook palette. I think both blue and green are good choices for the `Select` and you can base it on the current `Fill missing` or` Filter by` colors.

Reply:
<img width="720" alt="Screen Shot 2023-11-24 at 3 33 57 PM" src="https://github.com/livebook-dev/kino_explorer/assets/16871737/1c1f0934-217b-421f-a919-bf9bc21a966f">

How about emerald?